buildSrc/src/main/kotlin/ort-server-kotlin-jvm-conventions.gradle.kts

```diff
@@ -37,6 +37,7 @@ plugins {
 }
 
 dependencies {
+    testImplementation(project(":utils:logging"))
```

Comment:
Have you checked whether this in turn should be removed from `testImplementation` in the actual JVM projects?